src/interfaces/api/site.ts

```diff
@@ -284,8 +292,12 @@ export class EditSite {
   @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
   @Expose()
   default_post_listing_type: Option<string>;
+  @Transform(({ value }) => toOption(value), { toClassOnly: true })
+  @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
+  @Expose()
+  hide_modlog_mod_names: Option<boolean>;
   auth: string;
-
+  
```

Comment:
If you can't get husky working, at least manually run prettier, as it will clean up stuff like this.